translations/dutch.json

```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "DATA COMMANDO'S",
 		"DRIVER COMMANDS": "DRIVER COMMANDO'S",
 		"MODERATOR COMMANDS": "MODERATOR COMMANDO'S",
-		"LEADER COMMANDS": "LEADER COMMANDO'S",
+		"ADMIN COMMANDS": "LEADER COMMANDO'S",
```

Comment:
ADMINISTRATOR COMMANDO'S

(line 21 uses administrator in the translation + the other ranks are also directly copied over from english)